pkg/apis/config/v1alpha2/meshrootcertificate.go

```diff
@@ -123,9 +123,25 @@ type TresorCASpec struct {
 type MeshRootCertificateIntent string
 
 // MeshRootCertificateComponentStatus specifies the status of the certificate component,
-// can be (`True`, `False`, `Unknown`).
+// can be (`Issuing`, `Validating`, `Unknown`).
 type MeshRootCertificateComponentStatus string
```

Comment:
Should there also be an `error` status for `MeshRootCertificateComponentStatus`?